composer.json

```diff
@@ -29,14 +29,18 @@
     "ext-pdo": "*",
     "ext-simplexml": "*",
     "propel/propel1": "^1.6.8",
-    "sensio/framework-extra-bundle": "^5.2",
-    "symfony/framework-bundle": "^3.4"
+    "sensio/framework-extra-bundle": "^5.5",
+    "symfony/console": "^4.4",
+    "symfony/debug": "^4.4",
```

Comment:
Shouldn't this be a dev-dependency?

Reply:
It probably should indeed, I'll check. +1 Also, as I've done in other PRs, I'll add ^4.4 using a logical OR instead to remain compatible with 3.4 as well.